src/hooks/useSaveAudio/cleanUpImages.ts

```diff
@@ -53,10 +51,4 @@ export async function cleanUpImages() {
   console.log(
     `Deleted ${unlinkedImages.length} unlinked images in ${((performance.now() - start) / 1000).toFixed(4)}s.`,
   );
-
-  if (unlinkedImages.length > 0) {
-    queryClient.invalidateQueries({
-      queryKey: settingKeys.storageRelation("image-save-status"),
-    });
-  }
```

Comment:
I think some better phrasing is to say it's not really necessary - `saveArtwork()` will refresh all queries essentially (and runs right before this), so this invalidation isn't needed.

In addition, image cleanup isn't related to the "image save status" feature.